compiler-rt/lib/builtins/multc3.c

```diff
@@ -15,8 +15,6 @@
 #include "int_lib.h"
 #include "int_math.h"
 
-#if defined(CRT_HAS_TF_MODE)
```

Comment:
Is this diff still needed? The macro should be defined with the change to int_types.h. Or is the problem here that int128 is not supported?

Reply:
int128 isn't supported on z/OS yet.  This change enables quad precision without needing to have TF mode enabled.  We need this diff otherwise we don't get __multc3 defined.